src/components/app/CompareHome.js

```diff
@@ -81,7 +81,7 @@ class CompareHomeImpl extends PureComponent<Props, State> {
             value={profile2}
           />
           <input
-            className="compareHomeSubmitButton"
+            className="compareHomeSubmitButton photon-button photon-button-primary"
```

Comment:
:+1:

Reply:
FYI I did the same in #3146 (just added a `font-size: inherit` to `compareHomeSubmitButton` too)